azerothcore/azerothcore-wotlk
# fix(Core/GUARD_RETURN): correct typo

## CHANGES PROPOSED:

In this commit: https://github.com/azerothcore/azerothcore-wotlk/commit/4dd9b92759ad7e2a7e8e4ffce83cb4d90b8afa0c

A `RETURN_GUARD` was defined, however it was used as `RETURN_GUAD` (typo, missing `R` in `GUARD`).

- I've renamed it to `GUARD_RETURN` as it was originally written in CMaNGOS (see https://github.com/cmangos/mangos-wotlk/commit/8ae67274a754f821f0bea6a5f771eab3df764a57) and fixed the typos.
- Fixed a variable name being `mutex` instead of `_ mutex`
- Also, I've modified the header credits.


## ISSUES ADDRESSED:

None that I'm aware of

## TESTS PERFORMED:

- Compiles
- **NOT** tested in game
- **NOT** tested the extractors

## HOW TO TEST THE CHANGES:

- Build again the core with the tools and extract the client data (see https://www.azerothcore.org/wiki/Extract-Client-Data)
- Run the core with the newly extracted data
- Access the game and have a look around (there is nothing specific to be checked here)

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
